morphic.js

```diff
@@ -9403,16 +9443,33 @@ MenuItemMorph.prototype.mouseEnter = function () {
         this.image = this.highlightImage;
         this.changed();
     }
+    if (this.doubleClickAction instanceof MenuMorph) {
+        if (!contains(this.parent.children, this.doubleClickAction)) {
+            this.parent.addChild(this.doubleClickAction);
+		}
+        this.doubleClickAction.popup(this.world(),
+                                     new Point(this.parent.bounds.right()-10,
```

Comment:
How were 10 and 4 picked? Are they what looks good, or are they also defined elsewhere?